config/custom-environment-variables.yaml

```diff
@@ -112,3 +112,7 @@ ecosystem:
     store: ECOSYSTEM_STORE
     # Badge service (needs to add a status and color)
     badges: ECOSYSTEM_BADGES
+    # List of module names, or objects { name, config } for instantiation to use in sd-setup
+    setup: SD_SETUP_PLUGINS
```

Comment:
The first parameter is a map of all the default modules. The second parameter defines which plugins to actually use.

If you have the `screwdriver-scm-github` in the first parameter, it won't be loaded unless you specify it in the setup list.

Reference: https://github.com/screwdriver-cd/build-bookend/blob/master/index.js#L84-L86